src/components/Modal/ModalContent.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import { zIndexModal } from '../../styles/designTokens';
+
+const ModalContent = styled.section`
```

Comment:
There is better way to do it. For ModalWrapper component define styles:
```
position: fixed;
display: flex;
top: 0;
bottom: 0:
left: 0;
right: 0;
align-items: center;
justify-content: center;
```
It will automatically center its content.